opensearchconfig.go

```diff
@@ -26,6 +26,8 @@ const (
 	AuthIAM = "iam"
 	// ENV variable to control whether traces are send to datadog
 	DD_TRACE_ENABLED = "DD_TRACE_ENABLED"
+	// Identify the name of the trace in datadog
+	Opensearch = "Opensearch"
```

Comment:
```suggestion
	Opensearch = "opensearch"
```

Can we use lowercase names please?